source/fundamentals/authentication.txt

```diff
@@ -142,6 +143,157 @@ string as follow:
 
    To learn more on specifying the default mechanism, see :ref:`csharp-authentication-default`.
 
+.. _csharp-mongodb-aws:
+
+MONGODB-AWS
+~~~~~~~~~~~
+
+.. note::
+
+   The ``MONGODB-AWS`` authentication mechanism is available only for
+   MongoDB deployments on MongoDB Atlas.
+
+The ``MONGODB-AWS`` authentication mechanism uses your Amazon Web Services
+Identity and Access Management (AWS IAM) credentials to authenticate your
+user. You can either specify your credentials directly on a ``MongoClientSettings`` object 
```

Comment:
Yes, that way is valid. I know that the C# engineers want us to encourage users to call `MongoClientSettings.FromConnectionString()`, which is probably why Dima used this method in his C# examples. (I think they're pushing MongoClientSettings because it offers a superset of the config options available in the connection string.)

Do you think this page needs to show a MongoClientSettings-less way of enabling AWS auth? It's almost identical to the examples here but passes the connection string straight to the MongoClient constructor instead of via MongoClientSettings.

Reply:
The way that the Java guide accomplishes this is by adding an admonition that links to an older version of the docs.  See the [note directly above the X.509 section](https://www.mongodb.com/docs/drivers/java/sync/current/fundamentals/auth/#x.509).

Since this is the first version to include MONGODB-AWS and therefore no prior versions to link to, perhaps the best solution is to go with whatever Dima prefers.